[0m11:20:39.234432 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000021365FAD310>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000021365C7F850>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000021365FD7490>]}


============================== 11:20:39.243814 | e7f408a2-3d4c-4abf-b54b-97153dfd6a79 ==============================
[0m11:20:39.243814 [info ] [MainThread]: Running with dbt=1.9.3
[0m11:20:39.244402 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\metin\\.dbt', 'debug': 'False', 'warn_error': 'None', 'log_path': 'logs', 'fail_fast': 'False', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'invocation_command': 'dbt init', 'introspect': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m11:21:32.734661 [debug] [MainThread]: Starter project path: C:\Users\metin\OneDrive\Documents\DBT_DEMO\my-dbt-env\Lib\site-packages\dbt\include\starter_project
[0m11:21:32.801198 [info ] [MainThread]: 
Your new dbt project "my_dbt_db" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m11:21:32.802195 [info ] [MainThread]: Setting up your profile.
[0m11:21:32.806870 [error] [MainThread]: Encountered an error:
Runtime Error
  No adapters available. Learn how to install an adapter by going to https://docs.getdbt.com/docs/connect-adapters#install-using-the-cli
[0m11:21:32.808375 [debug] [MainThread]: Command `dbt init` failed at 11:21:32.808375 after 53.70 seconds
[0m11:21:32.809766 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002135FA1A210>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002136538A650>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000021365DCEED0>]}
[0m11:21:32.809766 [debug] [MainThread]: Flushing usage events
[0m11:21:33.328395 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m11:21:47.231264 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002639411F810>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002639416E990>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002639416E850>]}


============================== 11:21:47.236793 | 563f06f7-e741-4c3c-944f-44562ad4926b ==============================
[0m11:21:47.236793 [info ] [MainThread]: Running with dbt=1.9.3
[0m11:21:47.238299 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'profiles_dir': 'C:\\Users\\metin\\.dbt', 'log_path': 'logs', 'debug': 'False', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'introspect': 'True', 'static_parser': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'target_path': 'None', 'invocation_command': 'dbt init', 'send_anonymous_usage_stats': 'True'}
[0m11:21:52.853394 [debug] [MainThread]: Starter project path: C:\Users\metin\OneDrive\Documents\DBT_DEMO\my-dbt-env\Lib\site-packages\dbt\include\starter_project
[0m11:21:52.875306 [info ] [MainThread]: 
Your new dbt project "test" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m11:21:52.877301 [info ] [MainThread]: Setting up your profile.
[0m11:21:52.884703 [error] [MainThread]: Encountered an error:
Runtime Error
  No adapters available. Learn how to install an adapter by going to https://docs.getdbt.com/docs/connect-adapters#install-using-the-cli
[0m11:21:52.889377 [debug] [MainThread]: Command `dbt init` failed at 11:21:52.888374 after 5.78 seconds
[0m11:21:52.890371 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000263941407D0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002638DB8A210>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000026394020D10>]}
[0m11:21:52.890371 [debug] [MainThread]: Flushing usage events
[0m11:21:53.402362 [debug] [MainThread]: An error was encountered while trying to flush usage events
